package.json

```diff
@@ -38,15 +38,17 @@
   },
   "dependencies": {
     "babel-plugin-transform-es2015-modules-commonjs": "^6.22.0",
+    "babel-traverse": "^6.22.1",
+    "babel-types": "^6.22.0",
+    "babylon": "^6.15.0",
     "codemirror": "^5.1.0",
-    "devtools-launchpad": "^0.0.39",
+    "devtools-launchpad": "^0.0.40",
```

Comment:
This needs to land first
https://github.com/devtools-html/devtools-core/pull/172

And I'll lkely want to update reps as well with this change